templates/NextflowTower/nextflow-launch.yaml

```diff
@@ -10,15 +10,21 @@ Parameters:
   MinvCpus:
     Type: Number
     Description: The number of min CPUs for Batch compute environments
-    MinValue: 1
+    MinValue: 0
     MaxValue: 16
-    Default: 1
+    Default: 0
```

Comment:
Does this mean that all Nextflow processes will need to have a number of CPUs specified?

Reply:
This is a scaling configuration which, i believe, will allow batch to scale down to 0 resources when no processes are assigned to this batch pipeline.   

As I understand it,  you don't configure this as a tower end user.   This is a shared batch environment,  Tower should allow you to assign your processes to this batch pipeline.

Reply:
When running a Nextflow process, you can set the number of CPUs and amount of memory to be used for that specific process https://www.nextflow.io/docs/latest/reference/process.html#process-cpus. In our current deployment, when you do not specify a number of CPUs it defaults to 1, but this change seems to make that default 0. I am wondering what the impact of that change will be when one does not specify the cpu directive in a process definition.